config/workflow_config.json

```diff
@@ -33,6 +33,13 @@
         "parameter_type": "datetime",
         "key_name": "end_time"
       }
+    ],
+    "parameter_relations": [

```

Comment:
Yes I agree, both with the 'constraints' naming and placing inside a parameter description. I would name `key_2` as `other_parameter`.

It would be more intuitive and you can control which of the two parameters is constraint, and therefore gets the warning. For instance, if `end_time` is not larger than `start_time` I only want a warning on the `end_time` parameter for the mapeditor case. In some other situation you would want both parameters to get a warning.